### PR TITLE
updated dph doc

### PR DESCRIPTION
**Pull request by shraddaa**

### CODE REVIEW

File: docs/interfaces/data_product_hub/discovery.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ A simple yet powerful way to find Data Products is provided by typing the name,
...
 
-- **Tier:** Describes how the data product is organized, whether it follows a medallion architecture, is source-aligned, or is structured around specific entities or consumer needs.
+- **Tier:** Describes how the Data Product is organized, whether it follows a medallion architecture, is source-aligned, or is structured around specific entities or consumer needs.
 
-- **Domain:** Specifies the business domain to which the data product belongs. Multiple domains can be selected at once to filter for the desired Data Product.
+- **Domain:** Specifies the business domain to which the Data Product belongs. Multiple domains can be selected at once to filter for the desired Data Product.
 
-- **Owner:** Filters can be applied by owner, allowing users to direct questions to the appropriate person or team. The owner's DataOS User ID is used to list the various owners in the Owner dropdown, and only a single owner can be selected at a time.
+- **Owner:** Filters can be applied by owner, allowing users to direct questions to the appropriate person or team. The owner's DataOS User ID is used to list the various owners in the owner dropdown, and only a single owner can be selected at a time.
```
Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/discovery.md:37]: Why have you small cased owner, just a question?

**Review by pyjs-tmdc — CHANGES_REQUESTED** [review]
@shraddaa I have put some comments on this page, resolve them. Once they are done I will merge to master

File: docs/interfaces/data_product_hub/core_concepts.md [hunk 2]
```diff
@@ -91,32 +91,58 @@ In the Data Product Hub, an output is a materialized dataset generated from the
 <figcaption><i>Output</i></figcaption>
 </center>
 
-## Semantic models
 
-A semantic model is a [data model](/resources/lens/overview/) derived from the [Lens](/resources/lens/). It is fed by the inputs to structure the data in a more meaningful way.
+## Data APIs
 
+A data API refers to an application programming interface (API) that allows applications to interact with and retrieve data from a data source, such as a database or data warehouse, in a structured and controlled manner. Data APIs are used to enable communication between software applications and databases, allowing developers to query, filter, and manipulate data without directly interacting with the underlying data storage systems. Within DataOS, one can create the data APIs using [Talos Stack](/resources/stacks/talos/).
```
[the remark this answers is not in the document]
Comment (shraddaa) [reply docs/interfaces/data_product_hub/core_concepts.md:97]: Generic, Like if user want to know what is a Data API in general.

File: docs/interfaces/data_product_hub/core_concepts.md [hunk 3]
```diff
@@ -91,32 +91,58 @@ In the Data Product Hub, an output is a materialized dataset generated from the
...
 
-## Data APIs
+## Tier
 
-## Quality
+Tier is a classification for Data Products based on their purpose, whether by medallion architecture, source-aligned, or based on specific entities or consumers. For example, in the 'Customer Churn Prediction' Data Product, its tier is labeled as `Business Certified` which basically means the Data Product is business oriented. Similarly, each organization can have its own predefined tiers. Tier is populated from the Data Product manifest file
```
[the remark this answers is not in the document]
Comment (shraddaa) [reply docs/interfaces/data_product_hub/core_concepts.md:101]: fixed

File: docs/interfaces/data_product_hub/discovery.md (hunk 1 above)
Comment (shraddaa) [reply docs/interfaces/data_product_hub/discovery.md]: fixed

**Pull request merged**